README.md

```diff
@@ -9,7 +9,22 @@ You can see details of [the project roadmap here](https://napari.org/roadmaps/in
 
 ## contributing
 
-Contributions are encouraged! Please read our [contributing guide](https://napari.org/dev/developers/contributing/documentation/index.html) to get started. Given that we're in an early stage, you may want to reach out on our [Github Issues](https://github.com/napari/docs/issues) before jumping in.
+Contributions are encouraged! Given that we're in an early stage, you may want to reach out on our [Github Issues](https://github.com/napari/docs/issues) before jumping in.
+It's nice to be able preview how your contribution will look on the web, so here we provide a brief summary of what's required for a local setup that will allow you to do so.
+Please read our [contributing guide](https://napari.org/dev/developers/contributing/documentation/index.html) for more comprehensive information about contributing documentation.
+
+### quickstart: local setup
+* __Check the prerequisites__
+    1. Create a clean Python environment (e.g., with conda).
+    1. In that environment, create a [development installation](https://napari.org/dev/developers/contributing/dev_install.html#dev-installation) of `napari` on your local machine, first forking and cloning the main `napari` project if you've not previously done so.
+    1. Fork this repository, and then clone your fork to your local machine. NB: you may want to name your fork e.g. `napari-docs` rather than just `docs`.
+    1. With your local clone's root folder as the working directory, and with the environment created in the first step activated, install the docs requirements with `python -m pip install -r requirements.txt`.
+* __Build__ locally
+    * If you're building on Windows, a few extra steps are required; you can follow [this guide](https://napari.org/stable/developers/documentation/index.html#building-the-documentation-on-windows).
+    * From the root of your local clone of this repository, run one of these:
+        * `make docs`, if your changes include the example gallery
```

Comment:
Should we mention `make docs-xvfb` here, or is that getting too detailed? 
It is certainly more convenient not to have napari windows popping up all over the place while it runs, but I guess if `xvfb` needs to be installed separately then perhaps this makes the instructions a little too long for the quickstart.

Reply:
I *think* that only works on linux and if you have xvfb installed and set up?
I know I've fought with this in various docker/singularity containers.

Reply:
@GenevieveBuckley @psobolewskiPhD shall I add a note about this possibility, something like a bullet at the same indentation level as the Windows note, saying something like...

"If you're building on Linux and have `xvfb` set up, you can avoid windows opening during the build by using `make docs-xvfb`"

...or do you prefer as-is?

Reply:
I think that's something that's best left for reading the actual docs. what's here is enough to get someone to a build and then they can explore the other `make` options